[2023-01-04 04:10:21,519] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: populate_crypto_values.get_crypto_values scheduled__2021-03-03T07:00:00+00:00 [queued]>
[2023-01-04 04:10:21,526] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: populate_crypto_values.get_crypto_values scheduled__2021-03-03T07:00:00+00:00 [queued]>
[2023-01-04 04:10:21,527] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-01-04 04:10:21,527] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-01-04 04:10:21,527] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-01-04 04:10:21,554] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): get_crypto_values> on 2021-03-03 07:00:00+00:00
[2023-01-04 04:10:21,561] {standard_task_runner.py:52} INFO - Started process 61755 to run task
[2023-01-04 04:10:21,694] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'populate_crypto_values', 'get_crypto_values', 'scheduled__2021-03-03T07:00:00+00:00', '--job-id', '90', '--raw', '--subdir', 'DAGS_FOLDER/populate_crypto_values.py', '--cfg-path', '/tmp/tmpnoegm31g', '--error-file', '/tmp/tmpzavwbbh4']
[2023-01-04 04:10:21,695] {standard_task_runner.py:80} INFO - Job 90: Subtask get_crypto_values
[2023-01-04 04:10:21,784] {task_command.py:370} INFO - Running <TaskInstance: populate_crypto_values.get_crypto_values scheduled__2021-03-03T07:00:00+00:00 [running]> on host xuxon-virtual-machine
[2023-01-04 04:10:21,893] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=populate_crypto_values
AIRFLOW_CTX_TASK_ID=get_crypto_values
AIRFLOW_CTX_EXECUTION_DATE=2021-03-03T07:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-03-03T07:00:00+00:00
[2023-01-04 04:10:23,087] {logging_mixin.py:115} INFO - Encountered errors while inserting rows: [{'index': 0, 'errors': [{'reason': 'invalid', 'location': 'snapshot_date', 'debugInfo': '', 'message': "Invalid date: '04-03-2021'"}]}, {'index': 1, 'errors': [{'reason': 'invalid', 'location': 'snapshot_date', 'debugInfo': '', 'message': "Invalid date: '04-03-2021'"}]}]
[2023-01-04 04:10:23,087] {python.py:173} INFO - Done. Returned value was: None
[2023-01-04 04:10:23,112] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=populate_crypto_values, task_id=get_crypto_values, execution_date=20210303T070000, start_date=20230104T071021, end_date=20230104T071023
[2023-01-04 04:10:23,193] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-01-04 04:10:23,257] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-01-04 05:27:08,981] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: populate_crypto_values.get_crypto_values scheduled__2021-03-03T07:00:00+00:00 [queued]>
[2023-01-04 05:27:09,194] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: populate_crypto_values.get_crypto_values scheduled__2021-03-03T07:00:00+00:00 [queued]>
[2023-01-04 05:27:09,194] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-01-04 05:27:09,195] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-01-04 05:27:09,195] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-01-04 05:27:09,211] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): get_crypto_values> on 2021-03-03 07:00:00+00:00
[2023-01-04 05:27:09,216] {standard_task_runner.py:52} INFO - Started process 9166 to run task
[2023-01-04 05:27:09,221] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'populate_crypto_values', 'get_crypto_values', 'scheduled__2021-03-03T07:00:00+00:00', '--job-id', '119', '--raw', '--subdir', 'DAGS_FOLDER/populate_crypto_values.py', '--cfg-path', '/tmp/tmpcnbzn7s0', '--error-file', '/tmp/tmpg4il1kas']
[2023-01-04 05:27:09,222] {standard_task_runner.py:80} INFO - Job 119: Subtask get_crypto_values
[2023-01-04 05:27:09,276] {task_command.py:370} INFO - Running <TaskInstance: populate_crypto_values.get_crypto_values scheduled__2021-03-03T07:00:00+00:00 [running]> on host xuxon-virtual-machine
[2023-01-04 05:27:09,342] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=populate_crypto_values
AIRFLOW_CTX_TASK_ID=get_crypto_values
AIRFLOW_CTX_EXECUTION_DATE=2021-03-03T07:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-03-03T07:00:00+00:00
[2023-01-04 05:27:10,669] {logging_mixin.py:115} INFO - New rows have been added.
[2023-01-04 05:27:10,670] {python.py:173} INFO - Done. Returned value was: None
[2023-01-04 05:27:10,695] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=populate_crypto_values, task_id=get_crypto_values, execution_date=20210303T070000, start_date=20230104T082709, end_date=20230104T082710
[2023-01-04 05:27:10,730] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-01-04 05:27:10,757] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
